Example/EpoxyExample/Views/LayoutGroups/MessageRowStackView.swift

```diff
@@ -0,0 +1,145 @@
+// Created by Tyler Hedrick on 1/27/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import EpoxyCore
+import EpoxyLayoutGroups
+import UIKit
+
+final class MessageRowStackView: BaseRow, EpoxyableView {
```

Comment:
nit: might be helpful to have a comment describing why this is implemented using stack views